spec/controllers/prison/dashboards_controller_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     context "when logged in" do
       before do
-        sign_in user
+        allow(controller).to receive(:current_user).and_return(user)
```

Comment:
We aren't signing in though, we are just stubbing out `current_user` so would prefer a different name. How about `stub_current_user` ?